decidim-forms/spec/commands/decidim/forms/answer_questionnaire_spec.rb

```diff
@@ -169,7 +169,7 @@ def tokenize(id)
                 },
                 {
                   title: "CSV document",
```

Comment:
As this tests the "invalid" extensions, I believe this change is unnecessary, unless it is an admin who uploads the file.

Anyways, the change itself is OK, but I would reflect the change in the title of the document:
```suggestion
                  title: "Text document",
```